Detectors/TPC/simulation/test/testTPCDigitContainer.cxx

```diff
@@ -29,34 +30,32 @@ namespace TPC {
   BOOST_AUTO_TEST_CASE(DigitContainer_test1)
   {
     const SAMPAProcessing& sampa = SAMPAProcessing::instance();
+    static FairRootManager *mgr = FairRootManager::Instance();
     auto *mDigitContainer = new DigitContainer();
 
-    /// \todo add test with FairLink stuff if possible
-    std::vector<int> hitID = {1, 250, 3, 62, 1000};
-    //std::vector<int> MCevent = {1, 250, 3, 62, 1000};
-    //std::vector<int> MCtrack = {22, 3, 4, 23, 523};
+    std::vector<int> MCevent = {1, 250, 3, 62, 1000};
+    std::vector<int> MCtrack = {22, 3, 4, 23, 523};
     std::vector<int> CRU     = {1, 23, 36, 53, 214};
     std::vector<int> Time    = {231, 2, 500, 230, 1};
     std::vector<int> Row     = {12, 5, 6, 2, 6};
     std::vector<int> Pad     = {1, 15, 14, 23, 5};
     std::vector<int> nEle    = {60, 100, 250, 1023, 2};
 
     for(int i=0; i<CRU.size(); ++i) {
-      //mDigitContainer->addDigit(MCevent[i], MCtrack[i], CRU[i], Time[i], Row[i], Pad[i], nEle[i]);
-      mDigitContainer->addDigit(hitID[i], CRU[i], Time[i], Row[i], Pad[i], nEle[i]);
+      mgr->SetEntryNr(MCevent[i]);
+      mDigitContainer->addDigit(MCtrack[i], CRU[i], Time[i], Row[i], Pad[i], nEle[i]);
     }
 
     auto *mDigitsArray = new TClonesArray("o2::TPC::DigitMC");
```

Comment:
here it does not work. In the next line, the function fillOutputContainer requests a pointer, which is owed to the internal handling of the TClonesArrays in FairRoot (usually the mDigitsArray is what is registered to the FairRootManager)